src/Data/NonEmpty.purs

```diff
@@ -54,7 +60,7 @@ oneOf :: forall f a. (Alternative f) => NonEmpty f a -> f a
 oneOf (NonEmpty a fa) = pure a <|> fa
 
 instance showNonEmpty :: (Show a, Show (f a)) => Show (NonEmpty f a) where
-  show (NonEmpty a fa) = "(NonEmpty " ++ show a ++ " " ++ show fa ++ ")"
+  show (NonEmpty a fa) = "NonEmpty (" ++ show a ++ ") (" ++ show fa ++ ")"
```

Comment:
I've been trying to move my new libraries to this style because it's more consistent. In the absence of something like `showsPrec`, you end up adding too many parens on atomic things like String and Number.